### DISCUSSION TITLE
Please add a "Shreddit" function to the app!

### DISCUSSION THREAD

**Discussion by secretmango** [Docile-Alligator/Infinity-For-Reddit#962] [Ideas]
### The Problem
Its good that you know that Reddit "only" stores the last version of a post or comment you wrote.
But that one gets stored forever, even though you want to delete it. 

Thats why "Shreddit" e.g. exist, its a script that replaces every comment or post older than for example 10 days with some text or just a "0" or ".", and THEN deletes it.

---
Doing that manually is really time consuming.... I love Infinity4Reddit, as it has soo many great features.

A Shreddit integration would make the OpenSource-App more Privacy-Focused!

## Concept
A settings menu under privacy, to select
- the amount of days after which a post gets shredded
- what should be inserted
- exeptions, for example saved posts or a way to manually select posts not to be shredded

----

Thank you for your work!

**Comment by KSPAtlas** [comment]
Would this be susceptible to reveddit, which is built into Infinity?

**Reply by MysteriousOrb2000** [reply]
No, reveddit pulls the data from pushshift.

**Comment by MysteriousOrb2000** [comment]
This is definitely needed since not everyone is able run shreddit on the desktop.
